pkg/admissioncontroller/webhooks/admission/seedrestriction/admission.go

```diff
@@ -260,6 +263,49 @@ func (h *handler) admitSecret(ctx context.Context, seedName string, request admi
 		return h.admit(seedName, shoot.Spec.SeedName)
 	}
 
+	// Check if the secret is a bootstrap token for a ManagedSeed.
+	if strings.HasPrefix(request.Name, bootstraptokenapi.BootstrapTokenSecretPrefix) && request.Namespace == metav1.NamespaceSystem {
```

Comment:
What do you think about checking the secret content itself, especially that no extra groups via `auth-extra-groups` are added?

Reply:
Valid point, thanks, done with 8abd43f